lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -54,9 +54,12 @@ describe('asset-saver helper', () => {
     it('trace file saved to disk with only trace events', () => {
       const traceFilename = 'the_file-0.trace.json';
       const traceFileContents = fs.readFileSync(traceFilename, 'utf8');
-      assert.deepStrictEqual(JSON.parse(traceFileContents), {traceEvents});
+      const traceEventsFromDisk = JSON.parse(traceFileContents).traceEvents;
+      traceEventsFromDisk.forEach((evt, i) => {
+        assert.deepStrictEqual(evt, traceEvents[i]);
+      });
       fs.unlinkSync(traceFilename);
-    });
+    }).timeout(120000);
```

Comment:
according to https://travis-ci.org/GoogleChrome/lighthouse/jobs/327029200#L1499 the forEach takes just 160ms.

so we can nuke this timeout.